internal/cmd/io_binary.go

```diff
@@ -273,7 +278,7 @@ func (p *pipe) Read(b []byte) (int, error) {
 }
 
 func (p *pipe) Close() error {
-	p.l.Close()
+	_ = p.l.Close()
```

Comment:
I think we'd generally always like to just log on errors instead of doing this (I know it's to please the eventual new linter). I'm picturing if we ever turned off error checking on the linter and then we just have a handful of these floating around that we'll never remember hahah

Reply:
logged with a `TODO` context.